vocode/streaming/agent/base_agent.py

```diff
@@ -191,7 +202,7 @@ async def handle_generate_response(
             )
         # TODO: implement should_stop for generate_responses
         agent_span.end()
-        return False
+        return False, function_call
```

Comment:
why is the function call being returned here instead of just added to the action queue?

the text response is put into the queue above